### ISSUE TITLE
Android - Video frame is not visible on the screen

### ISSUE THREAD

**Issue by jay-s-kipl** [GetStream/stream-chat-react-native#1896]
**Describe the bug**
 I am using the default "MessageList" component. When I tap on any video attached in the chat, the video previewer is opened, but the video frame is not visible and is hidden in a corner at the top left.

**Dev environment info:**
- stream-chat-react-native - v5.6.0
- react-native - v0.66.5
- Device/Emulator - Android

**To Reproduce**
1. Go to any chat screen on an Android device.
2. Tap on any attached video.
3. Notice how the video is not visible, and when you try to drag it from the top left corner of the screen, you might see a small video frame.

**Expected behaviour**
Video should be properly visible on the screen.

**Video**

https://user-images.githubusercontent.com/101381898/209907135-76e94942-c3a8-4378-be95-b8b145bdbb23.mp4

**Comment by khushal87** [comment]
Copy of #1880

**Comment by jay-s-kipl** [comment]
Hi @khushal87,
Thanks for your response on #1880. I've checked out and followed the link to the https://github.com/GetStream/stream-chat-react-native/issues/1495, and the comment which suggests the fix, but this issue still persists at my end.

I've tried testing this on multiple devices which include -

Android Emulator - Pixel XL (Both Android 11 & 12), Pixel 4 (Android 12)
Samsung Galaxy A31 - Android 12
OnePlus 8T - Android 12
...and a few others too.
